x-pack/legacy/plugins/apm/public/components/app/Settings/CustomizeUI/CustomLink/CustomLinkFlyout/helper.ts

```diff
@@ -56,7 +56,9 @@ export const convertFiltersToArray = (customLink?: CustomLink): Filters => {
  */
 export const convertFiltersToObject = (filters: Filters) => {
   const convertedFilters = Object.fromEntries(
-    filters.filter(([key, value]) => !isEmpty(key) && !isEmpty(value))
+    filters
+      .filter(([key, value]) => !isEmpty(key) && !isEmpty(value))
+      .map(([key, value]) => [key, value.split(',').map(v => v.trim())])
```

Comment:
1. please write a test for `convertFiltersToObject `
2. perhaps change to:

```suggestion
      .map(([key, value]) => [key, value.split(',').filter(v => v.trim())])
```

input: `"a, "`
output: `['a']`

instead of:
output: `['a', '']`

Reply:
I've started writing tests for CustomLinkFlyout/helper.ts yesterday 😉